packages/components/BREAKING_CHANGES_LOG.md

```diff
@@ -1,5 +1,15 @@
 # Breaking changes log
 
+## vLAST
```

Comment:
vLast ?

Reply:
Waiting for approvals to fill it correctly :)